packages/sanity-cms-loader/src/index.ts

```diff
@@ -0,0 +1,284 @@
+import DataLoader, { Options } from 'dataloader';
+import { createClient, SanityClient } from '@sanity/client';
+import { map, partition } from 'lodash';
+import { promises as fs } from 'fs';
+import { join, dirname } from 'path';
+import * as ts from 'typescript';
+import { getWinstonLogger } from '@last-rev/logging';
+import Timer from '@last-rev/timer';
+import { ItemKey, ContentfulLoaders, FVLKey, RefByKey } from '@last-rev/types';
+import LastRevAppConfig from '@last-rev/app-config';
+
+const convertSanityDoc = (doc: any) => {
+  if (!doc) return null;
+  const { _id, _type, _updatedAt, ...fields } = doc;
+  return {
+    sys: { id: _id, updatedAt: _updatedAt, contentType: { sys: { id: _type } } },
+    fields
+  };
+};
+
+const logger = getWinstonLogger({ package: 'sanity-cms-loader', module: 'index', strategy: 'Cms' });
+
+const loadSchema = async (filePath: string): Promise<any> => {
+  const source = await fs.readFile(filePath, 'utf8');
+  const { outputText } = ts.transpileModule(source, {
+    compilerOptions: { module: ts.ModuleKind.CommonJS }
+  });
+  const module = { exports: {} } as any;
+  const fn = new Function('exports', 'require', 'module', '__filename', '__dirname', outputText);
+  fn(module.exports, require, module, filePath, dirname(filePath));
+  return module.exports.default || module.exports;
+};
+
+const mapSanityField = (field: any): any => {
```

Comment:
We talked about not reating the default fields in the schema and leaving everything up to the typedefs

Reply:
But the AI just did it 🤷 It uses the local file models in dev and the actual response from the API in prod.